package.json

```diff
@@ -519,7 +529,7 @@
     "zlib": "^1.0.5"
   },
   "engines": {
-    "node": ">=14.20.1 <19",
+    "node": ">=14.20.1 <21",
```

Comment:
Are we still supporting < 20?

Reply:
some light testing seems like it technically can. at least no worse than it was before.

however, i believe for 20 < it will become not compatible

Reply:
@ashwin-pc We're good to drop compatibility for node < 20, right? I think its more preferrable to remove and add back in later if necessary vs keep and have to maintain for all of 3.x (even if we can validate its fine through node 14 for now).

Reply:
I agree. Lets remove it.